📝 docs: add agents md

rivertwilight opened the pull request:
#### 💻 变更类型 | Change Type

<!-- For change type, change [ ] to [x]. -->

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 👷 build
- [ ] ⚡️ perf
- [ ] ✅ test
- [x] 📝 docs
- [ ] 🔨 chore

#### 🔀 变更说明 | Description of Change

Add agents.md in root directory.

#### 📝 补充信息 | Additional Information

See [agents.md](https://agents.md) for more details.

rivertwilight committed:
build: Add agents.md

arvinxx commented on AGENTS.md:
```diff
@@ -0,0 +1,133 @@
+## Best Practices
+
+- **Conservative for existing code, modern approaches for new features**
+- **Code Language**: Use Chinese for files with existing Chinese comments, American English for new files\*\*
```
\*\* 为啥是这样的

rivertwilight committed:
feat: Add agents md

tjx666 commented on AGENTS.md:
```diff
@@ -0,0 +1,133 @@
+- **Database**: PostgreSQL, PGLite, Drizzle ORM
+- **Testing**: Vitest, Testing Library
+- **Package Manager**: pnpm (monorepo structure)
+- **Build Tools**: Turbo, Vite
```
1. 严格来讲没有用 vite， 硬要说哪里用了 vite， 也是 vitest 内部用了 vite
2. build tools 应该说开发环境 nextjs 目前是 turbopack, 生产环境 webpack

tjx666 commented on AGENTS.md:
```diff
@@ -0,0 +1,133 @@
+- Use `pnpm` as the primary package manager
+- Use `bun` to run npm scripts
+- Use `bunx` to run executable npm packages
+- Use `pnpm dlx turbo run where <project_name>` to jump to packages
```
项目没有用 turborepo 吧

tjx666 commented on AGENTS.md:
```diff
@@ -0,0 +1,133 @@
+
+#### TypeScript
+
+- Follow strict TypeScript practices
```
这句话说了和没说一样

claude[bot] committed:
📝 docs: address tjx666's review comments on AGENTS.md

- Fix build tools description to reflect actual Next.js setup (Turbopack/Webpack)
- Remove incorrect Turborepo reference
- Improve TypeScript practices description for clarity

Co-authored-by: René Wang <RiverTwilight@users.noreply.github.com>
